File: testprojects/c#/benchmark/presentation/gui/linkedin/JobGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.linkedin;
using CSharpBenchmark.presentation.exception;

namespace CSharpBenchmark.presentation.gui.linkedin
{
    //Functional requirement 4.4
    //Test case 200: Exceptions in class presentation.gui.linkedin.JobGUI are only allowed when data is exchanged in Data Transfer Objects
    //Result: TRUE
    public class JobGUI
    {

        public JobGUI()
        {
            LinkedInDTO transferAccount = new LinkedInDTO();
            transferAccount.setName("Mathew");
            transferAccount.setPassword("Schilke");

            try
            {
                throw new LinkedInException(transferAccount);
            }
            catch (LinkedInException e)
            {
                Console.WriteLine(e.StackTrace);
            }
        }
    }
}

File: testresources/java/benchmark_2012/presentation/gui/linkedin/JobGUI.java
package presentation.gui.linkedin;

import javax.swing.JFrame;

import presentation.exception.LinkedInException;
import domain.linkedin.LinkedInDTO;
//Functional requirement 4.4
//Test case 200: Exceptions in class presentation.gui.linkedin.JobGUI are only allowed when data is exchanged in Data Transfer Objects
//Result: TRUE
public class JobGUI extends JFrame {

	public JobGUI(){
		LinkedInDTO transferAccount = new LinkedInDTO();
		transferAccount.setName("Mathew");
		transferAccount.setPassword("Schilke");	
		
		try {
			throw new LinkedInException(transferAccount);
		} catch (LinkedInException e) {
			e.printStackTrace();
		}
	}
}